app/assets/javascripts/application.js

```diff
@@ -14,3 +14,6 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+//= require jquery
```

Comment:
You've required jquery twice here. Notice the one at the top of the list. You should move `//= require bootstrap-sprockets` right under that.